client/src/components/v2/OnlineIndicator.js

```diff
@@ -0,0 +1,35 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Indicator from './Indicator';
+
+/**
+ * @typedef {object} OnlineIndicatorProps
+ * @property {string}           online State of the indicator
+ * @property {?string}          on Text to display for "online" status
+ * @property {?string}          off Text to display for "offline" status
+ */
+
+/**
+ * Indicator for Online / Offline
+ *
+ * @param {OnlineIndicatorProps} props Props
+ * @returns {React.Component<OnlineIndicatorProps>} Component
+ */
+export default function OnlineIndicator({ online, on, off }) {
```

Comment:
What is the point of the on and off props? I get that you did it so that the text is customisable, but I don't think we'll ever need to do that. I would rather just hard code it into line 22 for now so we don't have unnecessary props.

Reply:
If you feel strongly about the customisation of text, I would have called the props `onText` or `onMessage` etc. I had no idea what they were until I read the above docs.

Reply:
The props are optional so it doesn't really interfere with the interface. My thought process is that it makes `OnlineIndicator` more versatile. For example, we might want an indicator which has no text (just green/red light).